app/uk/gov/hmrc/sdes/bulkdownload/domain/Registration.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.sdes.bulkdownload.domain
+
+import play.api.libs.json.{Json, OFormat}
+
+case class Registration(serviceName: String, serviceUrl: String, metadata: Option[Map[String, String]] = None)
+
```

Comment:
Why put Registration case class in the domain package than to define it  with ServiceLocator, since Registration is not the subject of this microservice,